exchange/node_management_status.go

```diff
@@ -12,7 +12,7 @@ func GetNodeManagementPolicyStatus(ec ExchangeContext, orgId string, nodeId stri
 	glog.V(3).Infof("Getting node management policy status for node %v/%v and policy %v", orgId, nodeId, policyName)
 
 	var resp interface{}
-	resp = new(exchangecommon.NodeManagementPolicyStatus)
+	resp = new(exchangecommon.ExchangeNMPStatus)
```

Comment:
Because you are using `exchangecommon.ExchangeNMPStatus` now, please add it to the InvokeExchange() function in rpc.go: https://github.com/open-horizon/anax/blob/master/exchange/rpc.go#L309